frontend/app/src/components/history/transactions/TransactionContent.vue

```diff
@@ -728,6 +770,18 @@ watch(loading, async (isLoading, wasLoading) => {
         :edit="editableItem"
       />
     </big-dialog>
+
+    <big-dialog
+      :display="openAddTxDialog"
+      :title="tc('transactions.dialog.add_tx')"
+      :primary-action="tc('common.actions.save')"
+      :action-disabled="!txFormValid || txForm?.loading"
+      :loading="loading || txForm?.loading"
+      @confirm="confirmSaveTx()"
+      @cancel="clearAddTxDialog()"
+    >
+      <transaction-form ref="txForm" v-model="txFormValid" />
+    </big-dialog>
```

Comment:
How much do we actually need from the `TransactionContent` and how much can go to a new component?

Reply:
maybe we can move the dialog, like how i did it with ledger action

Reply:
Yes it would make more sense to keep the minimum footprint in TransactionContent